src/gocert.py

```diff
@@ -49,3 +85,29 @@ def is_initialized(self) -> bool:
             return False
         body = req.json()
         return body.get("initialized", False)
+
+    def create_first_user(self, username: str, password: str) -> int | None:
+        """Create the first admin user.
+
+        Args:
+            username: username of the first user
+            password: password for the first user. It must be longer than 7 characters, have at least one lowercase,
+                one uppercase and one number or special character.
+
+        Returns:
+            int | None: the id of the created user, or None if the request failed
+
+        """
+        try:
+            req = requests.post(
+                f"{self.url}/api/{self.API_VERSION}/accounts",
+                verify=self.ca_path if self.ca_path else None,
+                json={"username": username, "password": password},
+            )
+        except (requests.RequestException, OSError):
+            return None
+        if req.status_code != 201:
+            logger.warning("couldn't create first user: code %s, %s", req.status_code, req.text)
+            return None
+        id = req.json().get("id")
+        return int(id) if id else None
```

Comment:
The reason I avoided it in the first place was because I was thinking of handling the special 401 unauthorized case to log in again instead of throwing an error, but decided against that later on because it was unreadable. 

Updated the code to use this.